### PR TITLE
COMMERCE-12603

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12603

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 1]
```diff
@@ -246,6 +246,46 @@ definition {
 			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
 	}
 
+	@description = "COMMERCE-12603. As a admin, I want to filter products using Specification Facet widget and the URL updates accordingly"
+	@priority = 5
+	test AssertProductsAreFilteredBySpecificationFacetAndURLUpdatesAccordingly {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:251]: CanFilterProductsBySpecificationFacetAndURLUpdatesAccordingly

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:stop:ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:stop:stable

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by baharturk** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by jaydawu** [comment]
PR was forwarded manually here: https://github.com/brianchandotcom/liferay-portal/pull/142962

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**